marlowe-playground-client/src/Marlowe/Linter.purs

```diff
@@ -637,6 +637,22 @@ lintValue env t@(Term (MulValue a b) pos) = do
       markSimplification constToVal SimplifiableValue b sb
       pure (ValueSimp pos false t)
 
+lintValue env t@(Term (DivValue a b) pos) = do
+  sa <- lintValue env a
+  sb <- lintValue env b
+  case sa /\ sb of
+    (ConstantSimp _ _ v1 /\ ConstantSimp _ _ v2) -> pure (ConstantSimp pos true (v1 / v2))
```

Comment:
This would have to use the same as in the semantics instead of `/`, see `Scale` below:
```suggestion
    (ConstantSimp _ _ v) -> pure (ConstantSimp pos true (evalValue (makeEnvironment zero zero) (emptyState (Slot zero)) (S.DivValue (S.Constant v1) (S.Constant v2))))
```
Haven't checked whether this suggestion compiles